html-css/week2/index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Homework 2</title>
+    <link rel="stylesheet" href="./styles.css" />
+</head>
+
+<body>
+    <header id="header" class="header">
```

Comment:
Better to avoid using `id` entirely and replace it with second `class`. I think you didn't even use that `id`, so it's useless, delete it